plugin-api/pom.xml

```diff
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.sonarsource.sonarlint.core</groupId>
+    <artifactId>sonarlint-core-parent</artifactId>
+    <version>5.4-SNAPSHOT</version>
+  </parent>
+  <artifactId>sonarlint-plugin-api</artifactId>
+  <packaging>jar</packaging>
+  <name>SonarLint Plugin API</name>
+  <description>API used between SonarLint and analyzers</description>
+
+  <properties>
+    <jarsigner.skip>true</jarsigner.skip>
```

Comment:
Not sure about this, this might be required e.g for Eclipse (especially since `-core` has a dependency on this module).